CONTRIBUTING.md

```diff
@@ -0,0 +1,36 @@
+# Contributing
+
+## Conduct
+
+All contributors are expected to adhere to the project's [Code of Conduct](CODE_OF_CONDUCT.md).
+
+## Development
+
+To begin development on the VSCode extension for Swift you will need to install [Node.js](https://nodejs.org). On Linux, make sure to install Node.js from its official website or from [NodeSource](https://github.com/nodesource/distributions/) as the version included with your Linux distribution may be outdated.
+
+Next, clone this repository, and in the project directory run `npm install` to install all the dependencies.
+
+When you first open the project in VSCode you will be recommended to also install `ESLint` and `Prettier - Code formatter`. Please do so. We use these to ensure a consistent style and expect everyone who contributes to follow this style as well. 
+
+To run your version of the Swift extension while in VSCode, press `F5`. This will open up another instance of VSCode with it running. You can use the original version of VSCode to debug it. 
+
```

Comment:
Can we also mention how to unit test?

Either:
- From VS Code, by selection "Extension Tests" in the Run and Debug activity.
- Using `npm run test` from the command line (when VS Code is not running, or you'll get an error)